plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -647,13 +662,34 @@ private class DokkaDescriptorVisitor(
         return coroutineScope {
             val generics = async { descriptor.typeParameters.parallelMap { it.toVariantTypeParameter() } }
 
+            /**
+             * Workaround for problem with inheriting TagWrappers.
+             * There is an issue if one declare documentation in the class header for
+             * property using this syntax: `@property`
+             * The compiler will propagate it withing this tag to property and to its getters and setters.
+             *
+             * Actually, the problem impacts more of these tags, yet this particular tag was blocker for
+             * some opens-source plugin creators.
+             * TODO: Should rethink if we could fix it globally in dokka or in compiler itself.
```

Comment:
It could be fixed in the compiler. On the other hand, the compiler returns corresponding KDoc for some property etc. that is based on the positions of the original KDoc string, and therefore it is wrapped in the `Property` section, which is somehow correct. There could be some service in dokka to translate all of these, though one should rethink all the possibilities and cover them manually. Probably it won't be so hard afterall